frontend/packages/dev-console/src/components/buildconfig/sections/SourceSection.tsx

```diff
@@ -50,7 +58,14 @@ const SourceSection: React.FC<{}> = () => {
         />
       ) : null}
 
-      {type === 'git' ? <GitSection title="" formContextField="formData.source.git" /> : null}
+      {type === 'git' ? (
+        <GitSection
+          title=""
+          formContextField="formData.source.git"
+          gitUrlIsFromReload={gitUrlIsFromReload}
```

Comment:
This seems too specific for this use case in this particular form. Maybe you can add a generic value in formik state `reload` or `formReload`. It's a number and starts at `0` every time form reloads you increment this value. Have a separate `useEffect` for this and run the validation whenever the reload value changes. WDYT?

Reply:
changed to the counter :D